packages/reassure-cli/src/commands/init.ts

```diff
@@ -0,0 +1,130 @@
+import { copyFileSync, existsSync, readFileSync, appendFileSync } from 'node:fs';
+import { resolve } from 'node:path';
+import { logger } from '@callstack/reassure-logger';
+import type { CommandModule } from 'yargs';
+import {
+  CI_SCRIPT,
+  DANGERFILE_FALLBACK_JS,
+  DANGERFILE_FALLBACK_TS,
+  DANGERFILE_JS,
+  DANGERFILE_TS,
+  GIT_IGNORE,
+} from '../constants';
+import { applyCommonOptions, CommonOptions } from '../options';
+import { ASCII_BYE, ASCII_HELLO } from '../utils/ascii';
+import { configureLoggerOptions } from '../utils/logger';
+
+const TEMPLATE_PATH = `${__dirname}/../templates`;
+
+/**
+ * Generate requred Reassure files.
+ */
+export function run(options: CommonOptions): void {
+  configureLoggerOptions(options);
+
+  logger.color('brand', ASCII_HELLO);
+
+  setUpCiScript();
+  setUpDangerFile();
+  setUpGitIgnore();
+
+  logger.log('');
+  logger.color('brand', 'Finished initalizing new Reassure testing environment.');
+  logger.log('Please refer to our CI guide in order to set up your pipelines.');
+  logger.log('🔗 https://callstack.github.io/reassure/docs/installation#ci-setup');
+
+  logger.color('brand', ASCII_BYE);
+}
+
+export const command: CommandModule<{}, CommonOptions> = {
+  command: ['init'],
+  describe: 'Automates basic Reassure setup steps.',
+  builder: (yargs) => {
+    return applyCommonOptions(yargs);
+  },
+  handler: (args) => run(args),
+};
+
+function setUpCiScript() {
+  logger.log('');
+  logger.progress('#️⃣  CI Script:');
+
+  if (existsSync(CI_SCRIPT)) {
+    logger.clearLine();
+    logger.log(`✅  CI Script: skipping - already exists`);
+    logger.log(`🔗 ${resolve(CI_SCRIPT)}`);
+    return;
+  }
+
+  copyFileSync(`${TEMPLATE_PATH}/reassure-tests`, CI_SCRIPT);
+  logger.clearLine();
+  logger.log(`✅  CI Script: created`);
+  logger.log(`🔗 ${resolve(CI_SCRIPT)}`);
+}
+
+function setUpDangerFile() {
+  const [existingFile, fallbackFile] = queryDangerfile();
+
+  logger.log('');
+  logger.progress('#️⃣  Dangerfile:');
+
+  if (!existingFile) {
+    // If users does not have existing dangerfile, let use the JS one, as potentially less prolematic.
+    copyFileSync(`${TEMPLATE_PATH}/dangerfile`, DANGERFILE_JS);
+    logger.clearLine();
+    logger.log(`✅  Dangerfile: created`);
+    logger.log(`🔗 ${resolve(DANGERFILE_JS)}`);
+    return;
+  }
+
+  const existingContent = readFileSync(existingFile);
+  if (existingContent.includes('reassure')) {
+    logger.clearLine();
+    logger.log(`✅  Dangerfile: skipping - already contains Reassure code`);
+    logger.log(`🔗 ${resolve(existingFile)}`);
+    return;
+  }
+
+  logger.clearLine();
+  logger.log(`⚠️   Dangerfile: created ${fallbackFile} - merge with existing ${existingFile}`);
+  logger.log(`🔗 ${resolve(fallbackFile)}`);
+}
+
+function queryDangerfile(): [string, string] | [null, null] {
```

Comment:
Shouldn't we prompt the user whether they want to use Danger or not? Itcould include description of what it is

Reply:
Atm we do not support non-danger integrations, so Danger Setup is Linda necessary. When we add any alternative, we can prompt the user about it.

Reply:
You can use Reassure without comment reporter. It's only a recommended setup, but not strictly required to run the comparison on a CI